dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -2600,7 +2616,11 @@ class CXRefManager: public CXRefManagerBase
         Owned<IGroup> g;
         unsigned j;
         if (!nclusters) {
+#ifdef _CONTAINERIZED
```

Comment:
preferable to use isCotainerized()

In fact, I think it would be worth just changing the message to the planes version.
In BM, internally, we generate planes for all clusters, such that methods like getStoragePlane/getPlanesIterator work with clusters.

Reply:
Changed to planes message.